extensions/azure/data-plane/storage/src/main/java/org/eclipse/dataspaceconnector/azure/dataplane/azurestorage/pipeline/AzureStorageDataSink.java

```diff
@@ -33,7 +33,7 @@ public class AzureStorageDataSink extends ParallelSink {
     private String accountName;
     private String containerName;
     private String sharedKey;
-    private BlobAdapterFactory blobAdapterFactory;
+    private BlobStoreApi blobStoreApi;
```

Comment:
I guess  you removed BlobAdapterFactory to simplify the code? A quick explanation about why you removed the BlobAdapterFactory and the changes it implies in the PR description would have helped for the review.